code/utils.py

```diff
@@ -70,11 +67,8 @@ def get_image_paths(folder: Path) -> List[Path]:
     Returns:
         A list of the full paths to the images in the folder.
     """
-    return sorted([
-        folder.joinpath(f.name) for f in folder.iterdir() if ((
-            folder.joinpath(f.name).is_file()) and (".DS_Store" not in f.name))
-    ],
-                  key=str)
+    return sorted([folder.joinpath(f.name) for f in folder.iterdir() if
+                   ((folder.joinpath(f.name).is_file()) and (".DS_Store" not in f.name) and (f.suffix.casefold() in IMAGE_EXTS))], key=str)
```

Comment:
checking ".DS_Store" is not necessary here as it doesn't have image_exts anyway.

Reply:
We still should check in case valid images are found within the ".DS_Store" folder.

Reply:
It already checks if the entity is file, .is_file(), so I think that will not happen, what do you think?

Reply:
I was thinking of a file with name "/home/person/data/.DS_Store/image_1.png".